proto/xudrpc.proto

```diff
@@ -410,6 +424,21 @@ message Order {
   double hold = 10 [json_name = "hold"];
 }
 
+message Trade {
+  // The order id of the maker order involved in this trade.
+  string maker_order_id = 1 [json_name = "maker_order_id"];
+  // The order id of the taker order involved in this trade, if applicable.
+  string taker_order_id = 2 [json_name = "taker_order_id"];
+  // The payment hash of the swap that filled this trade, if applicable.
+  string r_hash = 3 [json_name = "r_hash"];
+  // The quantity transacted in this trade.
+  int64 quantity = 4 [json_name = "quantity"];
+  // pair id of trade
+  string pair_id = 5 [json_name = "pair_id"];
+  // local id of trade
```

Comment:
The trade itself doesn't have a local id, so this field can probably be dropped.